tools/RepoTasks/RepoTasks.RemoteWorker/AppDomainWorker.cs

```diff
@@ -161,61 +197,205 @@ internal Configuration BuildXmlConfiguration(IEnumerable<Type> cmdletTypList)
+                            {
+                                tableHeaders.Add(tableColumnHeader);
+                                tableColumnItems.Add(tableColumnItem);
+                            }
+                            else
+                            {
+                                columnHeadersWithPosition.Add(Tuple.Create(tableColumnHeader, position));
+                                columnItemsWithPosition.Add(Tuple.Create(tableColumnItem, position));
+                                dupTrackerPosition.Add(Tuple.Create(memberInfo.DeclaringType?.Name, memberInfo.DeclaringType?.BaseType?.Name, memberInfo.Name, position));
+                            }
+                        }
 
-                if (_onlyMarkedProperties && ps1XmlAttribute == null) throw new InvalidCastException("Ps1XmlAttribute");
+                        if ((ps1XmlAttribute.Target & ViewControl.List) == ViewControl.None) continue;
 
-                if (ps1XmlAttribute == null ||
-                    (ps1XmlAttribute.Target & ViewControl.Table) != ViewControl.None)
+                        var listItem = (ps1XmlAttribute.ScriptBlock != null)
+                            ? new ListItem
+                            {
+                                ScriptBlock = ps1XmlAttribute.ScriptBlock,
+                            }
+                            : new ListItem
+                            {
+                                PropertyName = memberInfo.Name,
+                            };
+
+                        if (position == Ps1XmlConstants.DefaultPosition)
+                        {
+                            listItems.Add(listItem);
+                        }
+                        else
+                        {
+                            listItemsWithPosition.Add(Tuple.Create(listItem, position));
+                        }
+                    }
+                }
+                else
                 {
                     tableHeaders.Add(new TableColumnHeader
                     {
-                        Label = ps1XmlAttribute?.Label ?? memberInfo.Name
+                        Label = memberInfo.Name
                     });
 
                     tableColumnItems.Add(new TableColumnItem
                     {
                         Alignment = Alignment.Left,
                         PropertyName = memberInfo.Name
                     });
-                }
 
-                if (ps1XmlAttribute == null ||
-                    (ps1XmlAttribute.Target & ViewControl.List) != ViewControl.None)
-                {
                     listItems.Add(new ListItem
                     {
                         PropertyName = memberInfo.Name
                     });
                 }
             }
+
+            // track duplicate
+            if (dupTrackerGroupBy.Count > 1)
+            {
+                var warning = "Found a duplicate of property'GroupByThis'. 'GroupByThis=true' have the following properties: ";
```

Comment:
Nitpick : a space is needed between `property` and `GroupByThis`

What do you think of the wording :  _"`GroupByThis` is `true` for the following properties"_

Reply:
fixed